lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -287,6 +287,17 @@
                                     [tooltip]="getCellTooltip(row, col)">
                                 </adf-datatable-cell>
                             </div>
+                            <div *ngSwitchCase="'boolean'" [attr.tabindex]="data.getValue(row, col, resolverFn)? 0 : -1" class="adf-cell-value"
+                                 [attr.data-automation-id]="'text_' + data.getValue(row, col, resolverFn)">
+                                <adf-datatable-cell
+                                    [copyContent]="col.copyContent"
+                                    [data]="data"  
+                                    [column]="col"
+                                    [row]="row"
+                                    [resolverFn]="resolverFn"
+                                    [tooltip]="getCellTooltip(row, col)">
+                                </adf-datatable-cell>
```

Comment:
we should create a new component for handle any new specific type of data column (check date/json/filesize types in the code).
with different component for each type, we can better maintain, customize and improve in the future.
```suggestion
                                </adf-boolean-cell>
```